client/apps/edit2/components/header/index.jsx

```diff
@@ -170,17 +182,20 @@ export class EditHeader extends Component {
             {isDeleting ? 'Deleting...' : 'Delete'}
           </button>
 
-          <button
+          <SaveButton
             className='avant-garde-button'
-            style={{color: this.getSaveColor()}}
+            color={this.getSaveColor()}
             onClick={this.onSave}
           >
             {this.getSaveText()}
-          </button>
+          </SaveButton>
 
-          <a href={article.getFullSlug()} target='_blank'>
+          <a
+            href={`${sd.FORCE_URL}/article/${article.slug}`}
```

Comment:
I added `forceURL` to our redux store (https://github.com/artsy/positron/blob/master/client/reducers/appReducer.js#L8). 
As a rule of thumb, I think it's a good idea to limit usage of `sd` in components and try to limit its use to set up `initialState` in our reducers. That'll increase portability

Reply:
Sounds good, I'll update this now